sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -324,10 +324,15 @@ class AccountSasPermissions(object):
         Valid for the following Object resource types only: queue messages.
     :param bool process:
         Valid for the following Object resource type only: queue messages.
+    :param bool tag:
```

Comment:
Discussed with Anna offline:
"tag" should probably be "tags".
And these should all probably go into kwargs and be :keyword: